pkg/pillar/containerd/containerd.go

```diff
@@ -185,76 +185,98 @@ func (client *Client) CtrWriteBlob(ctx context.Context, blobHash string, expecte
 }
 
 //CtrUpdateBlobInfo updates blobs info
-func (client *Client) CtrUpdateBlobInfo(updatedContentInfo content.Info, updatedFields []string) error {
-	if _, err := client.contentStore.Update(client.ctrdCtx, updatedContentInfo, updatedFields...); err != nil {
+func (client *Client) CtrUpdateBlobInfo(ctx context.Context, updatedContentInfo content.Info,
+	updatedFields []string) error {
+	if ctx == nil {
+		ctx = client.ctrdCtx
+	}
```

Comment:
Do we need this flexibility as opposed to requiring the the caller pass a context?

Reply:
Removed.